ops/charm.py

```diff
@@ -18,6 +18,7 @@
 import logging
 import os
 import pathlib
+from typing import NoReturn  # Use Never in Python 3.11+
```

Comment:
That's where I originally had it :). I didn't notice it moved. It seems if you run `tox -e fmt` it moves it up - I assume because of the comment. I think it's better in the list than having the comment, so will adjust.